Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using UniRx;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using WIMW.Input;
+using Zenject;
+
+namespace WhereIsMyWife.Managers
+{
+    public partial class InputEventManager : IPlayerInputEvent
```

Comment:
Make this class `MonoBehavior` and a singleton.

Reply:
01914c8aa11b59b92a6815f015a1883194a1bf95